tests/register_test.js

```diff
@@ -0,0 +1,34 @@
+const base = require('../pages/base');
+const { passwordField } = require('../pages/register');
```

Comment:
Remove lines 1-2. CodeceptJS already imports objects as parameters in array function, row 15 and in config file